controllers/qrCodeAuth.js

```diff
@@ -0,0 +1,26 @@
+const qrAuthQuery = require("../models/qrCodeAuth");
+const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+
+const updateAuthStatus = async (req, res) => {
+  try {
+    const userId = req.userData.id;
```

Comment:
i don't think we are passing any data to the request

Reply:
we will be getting it from the token itself

Reply:
Yes you are right, we are not passing any data to request. But the thing is, in our authenticate middleware, we are storing userId on req.userData object, so i just referenced id on that object. 😅😅

Reply:
ok i checked , rn looks good then